testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -415,5 +415,8 @@
 
 GSD = resource_filename(__name__, 'data/example.gsd')
 
+DihedralsArray = resource_filename(__name__, 'data/adk_oplsaa_dihedrals.npy')
```

Comment:
You also need to add the to `__all__` (I only learned that from the test that failed!)